easybuild/easyblocks/o/openssl_wrapper.py

```diff
@@ -0,0 +1,268 @@
+        self.ssl_sysheader = None
+        self.ssl_sysengines = None
+
+        if not self.cfg.get('wrap_system_openssl'):
+            return
+
+        # Check the system libraries of OpenSSL
+        libssl = {
+            '1.0': {
+                DARWIN: 'libssl.1.0.dylib',
+                LINUX: 'libssl.so.10',
+            },
+            '1.1': {
+                LINUX: 'libssl.so.1.1',
+                DARWIN: 'libssl.1.1.dylib',
+            },
+        }
+
+        os_type = get_os_type()
+
+        if self.version in libssl and os_type in libssl[self.version]:
+            libssl_so = libssl[self.version][os_type]
+        else:
+            raise EasyBuildError("Don't know name of OpenSSL system library for version %s and OS type %s",
+                                 self.version, os_type)
+
+        try:
+            libssl_obj = ctypes.cdll.LoadLibrary(libssl_so)
+            self.log.info("Absolute path to %s: %s" % (libssl_so, libssl_obj))
+        except OSError:
+            self.log.info("Library '%s' not found in host system", libssl_so)
+        else:
+            # ctypes.util.find_library only accepts unversioned library names
+            if os_type == LINUX:
+                # find path to library with dlinfo
+                self.ssl_syslib = locate_solib(libssl_obj)
+            elif os_type == DARWIN:
+                # ctypes.macholib.dyld.dyld_find accepts file names and returns full path
+                self.ssl_syslib = ctypes.macholib.dyld.dyld_find(libssl_so)
+            else:
+                raise EasyBuildError("Unknown host OS type: %s", os_type)
+
+        if self.ssl_syslib:
+            self.log.info("Found library '%s' in: %s", libssl_so, self.ssl_syslib)
+        else:
+            self.log.info("Library '%s' not found!", libssl_so)
+
+        # Directory with engine libraries
+        if self.ssl_syslib:
+            lib_dir = os.path.dirname(self.ssl_syslib)
+            lib_engines_dir = [
+                os.path.join(lib_dir, self.openssl_engines[self.version]),
+                os.path.join(lib_dir, 'openssl', self.openssl_engines[self.version]),
+            ]
+
+            for engines_path in lib_engines_dir:
+                if os.path.isdir(engines_path):
+                    self.ssl_sysengines = engines_path
+                    self.log.debug("Found OpenSSL engines in: %s", self.ssl_sysengines)
+
+            if not self.ssl_sysengines:
+                self.ssl_syslib = None
+                print_warning("Found OpenSSL in host system, but not its engines."
+                              "Falling back to OpenSSL in EasyBuild")
+
+        # Check system include paths for OpenSSL headers
+        cmd = "gcc -E -Wp,-v -xc /dev/null"
+        (out, ec) = run_cmd(cmd, log_all=True, simple=False, trace=False)
+
+        sys_include_dirs = []
+        for match in re.finditer(r'^\s(/[^\0\n]*)+', out, re.MULTILINE):
+            sys_include_dirs.extend(match.groups())
```

Comment:
```suggestion
            sys_include_dirs.extend(match.groups())
        if self.version = '1.1':
            sys_include_dirs.append('/usr/include/openssl11')
```

Reply:
@Micket fixed in https://github.com/easybuilders/easybuild-easyblocks/pull/2429/commits/4101ba39804eca373611651a26fbf3aa56811e18